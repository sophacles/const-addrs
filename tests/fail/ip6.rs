use const_addrs::ip6;

fn main() {
    let _ = ip6!("2001:db8::32::23");
}
