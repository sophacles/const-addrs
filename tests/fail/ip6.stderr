error: invalid IPv6 address syntax
 --> tests/fail/ip6.rs:4:18
  |
4 |     let _ = ip6!("2001:db8::32::23");
  |                  ^^^^^^^^^^^^^^^^^^
